scipy/integrate/_quadpack_py.py

```diff
@@ -61,6 +61,10 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
         Non-zero to return a dictionary of integration information.
         If non-zero, warning messages are also suppressed and the
         message is appended to the output tuple.
+    complex_func : bool, optional
+        Indicate if the function's (func) return type is real
```

Comment:
Backticks in ReST are different than in markdown.
https://numpydoc.readthedocs.io/en/latest/format.html#parameters
```suggestion
        Indicate if the function's (`func`) return type is real
```
(In many cases the guidelines for backticks here are incomplete and/or disagreeable, but they are what they are.)